external/webL10n/l10n.js

```diff
@@ -998,7 +998,6 @@ document.webL10n = (function(window, document, undefined) {
       loadLocale(lang, function() {
         if (callback)
           callback();
-        translateFragment();
```

Comment:
Mention this change at https://github.com/yurydelendik/pdf.js/blob/6f76510a74f086de4fdb9a21a677649de9618c56/external/webL10n/l10n.js#L24 as well. If we update `webL10n` in the future, this change may be accidentally reverted if it's not mentioned there.